go.mod

```diff
@@ -1,25 +1,31 @@
 module github.com/marcboeker/go-duckdb
 
-go 1.23
+go 1.23.4
```

Comment:
Maybe switch to 1.24?

Reply:
Done: https://github.com/marcboeker/go-duckdb/pull/379.